04/app.js

```diff
@@ -5,4 +5,54 @@ const menuItems = [
     {text: 'start', url: '/'},
     {text: 'galeria', url: '/gallery'},
     {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+];
+
+const navElement = document.querySelector("nav");
+
+if(navElement){
+    const ulElement = document.createElement("ul");
+    navElement.appendChild(ulElement);
+    
+    for(let i = 0; i < 3; i++) {
+
+        const liElement = document.createElement("li");
+        ulElement.appendChild(liElement);
+
+        const linkElement = document.createElement("a");
+        liElement.appendChild(linkElement);
+
+        if(i === 0) {
```

Comment:
Tutaj też brak elastyczności. Jeśli byłoby 100 elementów to byś napisał 100 `if`-ów?
Lepiej `linkElement.innerText = menuItems[i].text` - wiążesz iterację z konkretnym elementem w tablicy.